src/huggingface_hub/hf_api.py

```diff
@@ -1713,6 +1841,13 @@ def upload_file(
             identical_ok (`bool`, *optional*, defaults to `True`):
                 Deprecated: will be removed in 0.11.0.
                 Changing this value has no effect.
+            commit_message (`str`, *optional*):
+                The summary / title / first line of the generated commit
+            commit_description (`str` *optional*)
+                The description of the generated commit
+            create_pr (`boolean`, *optional*):
+                Whether or not to create a Pull Request from `revision` with that commit.
+                Defaults to `False`.
 
         Returns:
             `str`: The URL to visualize the uploaded file on the hub
```

Comment:
I think we're changing the return value here? And it depends on `create_pr`?

Reply:
Addressed in 552bfb6